content/overview/installation/_index.en.md

```diff
@@ -39,3 +39,10 @@ You can find our packagecloud.io page [here](https://packagecloud.io/wasmCloud/c
    {{% /tab %}}
 {{% /tabs %}}
 
+### (MacOS) Install the Homebrew tap
+wasmCloud uses [Homebrew](https://brew.sh/) for our MacOS builds
+
+//TODO

```

Comment:
If this is no longer a todo, we should remove it from the doc.

Reply:
@billyoung Are the homebrew instructions ready? If not, I can remove it and we can add it later.

Reply:
@brooksmtownsend the instructions are here: https://github.com/wasmCloud/wasmcloud-dev-site/blob/main/content/overview/installation/_index.en.md

```
brew tap wasmcloud/wasmcloud
brew install wasmcloud wash
```
Just a reminder: there will not be a homebrew package for `wasmCloud` until we create a release

Reply:
Going to throw these in the install doc then, and we'll make sure to release it 😛